pkg/connector/matrixfmt/html.go

```diff
@@ -294,6 +294,7 @@ func (parser *HTMLParser) basicFormatToString(node *html.Node, ctx Context) *Ent
 	case "ol":
 		return str.Format(linkedinfmt.Style{Type: linkedinfmt.StyleList, Ordered: true})
 	case "li":
+		str = NewEntityString("- ").Append(str).AppendString("\n")
 		return str.Format(linkedinfmt.Style{Type: linkedinfmt.StyleListItem})
```

Comment:
Does linkedin actually have a list entity type in the formatting protocol? If not, probably need to clean up all of this to remove types that don't exist

Reply:
Wait, what? We can't send any formatting? Or they reject some of them while accepting others?